experiments/nlp_experiments/classification/bert_mcdropout.py

```diff
@@ -0,0 +1,180 @@
+import sys
+sys.path.append("/app/baal")
+import argparse
+import random
+from copy import deepcopy
+
+import torch
+import torch.backends
+from torch import optim
+from transformers import BertTokenizer
+from transformers import BertForSequenceClassification
+from tqdm import tqdm
+
+from baal.active import get_heuristic, ActiveLearningDataset
+from baal.active.active_loop import ActiveLearningLoop
+from baal import BaalHuggingFaceTrainer
+from experiments.nlp_experiments.classification import CSVClassificationDataset
+
+"""
+Minimal example to use BaaL for NLP Classification.
+"""
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--epoch", default=100, type=int)
+    parser.add_argument("--batch_size", default=32, type=int)
+    parser.add_argument("--initial_pool", default=1000, type=int)
+    parser.add_argument("--n_data_to_label", default=100, type=int)
+    parser.add_argument("--lr", default=0.001)
+    parser.add_argument("--heuristic", default="bald", type=str)
+    parser.add_argument("--iterations", default=20, type=int)
+    parser.add_argument("--shuffle_prop", default=0.05, type=float)
+    parser.add_argument('--learning_epoch', default=20, type=int)
+    return parser.parse_args()
+
+def get_datasets(initial_pool):
```

Comment:
We could use the hugging faces datasets module so that this example is even simpler. We can use the SST-2 dataset.